### PR TITLE
Add compat data RTCPeerConnectionIceEvent

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnectionIceEvent

### CODE REVIEW

File: api/RTCPeerConnectionIceEvent.json [hunk 1]
```diff
@@ -0,0 +1,203 @@
+{
+  "api": {
+    "RTCPeerConnectionIceEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnectionIceEvent",
+        "support": {
+          "webview_android": [
+            {
+              "alternative_name": "RTCIceCandidateEvent",
+              "version_added": true
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: Also add:
"version_removed": "56"

File: api/RTCPeerConnectionIceEvent.json [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+            },
+            {
+              "version_added": "56"
+            }
+          ],
+          "chrome": [
+            {
+              "alternative_name": "RTCIceCandidateEvent",
+              "version_added": true
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: Also add:
"version_removed": "56"

File: api/RTCPeerConnectionIceEvent.json [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+            },
+            {
+              "version_added": "56"
+            }
+          ],
+          "chrome_android": [
+            {
+              "alternative_name": "RTCIceCandidateEvent",
+              "version_added": true
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: Also add:
"version_removed": "56"

File: api/RTCPeerConnectionIceEvent.json [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "candidate": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnectionIceEvent/candidate",
+          "support": {
+            "webview_android": [
+              {
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: You don't need the alternate name info for the members.

File: api/RTCPeerConnectionIceEvent.json [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+                "alternative_name": "RTCIceCandidateEvent",
+                "version_added": true
+              },
+              {
+                "version_added": "56"
+              }
+            ],
+            "chrome": [
+              {
+                "alternative_name": "RTCIceCandidateEvent",
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: You don't need the alternate name info for the members.

File: api/RTCPeerConnectionIceEvent.json [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+                "version_added": true
+              },
+              {
+                "version_added": "56"
+              }
+            ],
+            "chrome_android": [
+              {
+                "alternative_name": "RTCIceCandidateEvent",
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: You don't need the alternate name info for the members.

File: api/RTCPeerConnectionIceEvent.json [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "RTCPeerConnectionIceEvent": {
+        "__compat": {
+          "description": "<code>RTCPeerConnectionIceEvent()</code>",
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: Add constructor to make it clearer:
    "description": "\<code>RTCPeerConnectionIceEvent()\</code> constructor",

File: api/RTCPeerConnectionIceEvent.json [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnectionIceEvent/RTCPeerConnectionIceEvent",
+          "support": {
+            "webview_android": [
+              {
+                "alternative_name": "RTCIceCandidateEvent",
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: You don't need the alternate name info for the members.

File: api/RTCPeerConnectionIceEvent.json [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+                "version_added": true
+              },
+              {
+                "version_added": "56"
+              }
+            ],
+            "chrome": [
+              {
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: You don't need the alternate name info for the members.

File: api/RTCPeerConnectionIceEvent.json [hunk 10]
```diff
@@ -0,0 +1,203 @@
...
+                "alternative_name": "RTCIceCandidateEvent",
+                "version_added": true
+              },
+              {
+                "version_added": "56"
+              }
+            ],
+            "chrome_android": [
+              {
+                "alternative_name": "RTCIceCandidateEvent",
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: You don't need the alternate name info for the members.

File: api/RTCPeerConnectionIceEvent.json [hunk 11]
```diff
@@ -0,0 +1,203 @@
...
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": null
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: true

File: api/RTCPeerConnectionIceEvent.json [hunk 12]
```diff
@@ -0,0 +1,203 @@
...
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: true

File: api/RTCPeerConnectionIceEvent.json [hunk 13]
```diff
@@ -0,0 +1,203 @@
...
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": null
```
Comment (teoli2003) [inline api/RTCPeerConnectionIceEvent.json]: true

File: api/RTCPeerConnectionIceEvent.json (hunk 1 above)
Comment (bunnybooboo) [reply api/RTCPeerConnectionIceEvent.json]: Ahh yeah, of course! :slightly_smiling_face:

**Pull request merged**